docs/AgentNodePlugin.md

```diff
@@ -2,13 +2,40 @@
 
 This plugin allows you to run the terraform stages in a docker container. This **DOES NOT WORK WITH AWSSUME** you should be using iam_block with aws provider with terraform.
 
+### Using pre-build docker image
 ```
 // Jenkinsfile
 @Library(['terraform-pipeline']) _
 
 Jenkinsfile.init(this)
 
-AgentNodePlugin.withAgentDockerImage('hashicorp/terraform:0.10.2')
+def terraformVersion = TerraformPlugin.checkVersion()
```

Comment:
Is this pertinent to the AgentNodePlugin?

Reply:
that isn't pertinent. I can remove that feature but I wanted to show. how to use use it with the terraform version. Cause just having a terraform version file won't pull the right image.